lib/graphql/resolvers.js

```diff
@@ -131,7 +145,8 @@ const staticConfig = ({ currentConfigVersion, deviceId, deviceName, pq, settings
         }),
         _.update('triggersAutomation', _.mapValues(_.eq('Automatic'))),
         addOperatorInfo(operatorInfo),
-        addReceiptInfo(receiptInfo)
+        addReceiptInfo(receiptInfo),
+        _.update('screenOptions', addMachineScreenOpts(machineScreenOpts))
```

Comment:
Isn't this `_.update()` unnecessary here? It's already in `addMachineScreenOpts()`.